ui/package.json

```
@@ -10,20 +10,28 @@
   },
   "dependencies": {
     "@apollo/client": "4.0.1",
+    "@iarna/toml": "^2.2.5",
```

Let's use strict versions to avoid unexpected  behaviours

```suggestion
    "@iarna/toml": "2.2.5",
```